cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AggregatingFunctionsTest.scala

```diff
@@ -84,88 +84,96 @@ class AggregatingFunctionsTest extends DocumentingTest {
     p("The following graph is used for the examples below:")
     graphViz()
     section("avg()", "functions-avg") {
-      p("`avg()` calculates the average of a numeric column.")
-      function("avg(expression)", ("expression", "A numeric expression."))
+      p(
+        "`avg()` returns the average of a set of numeric values.")
+      function("avg(expression)", "Either an Integer or a Float, depending on the values returned by `expression` and whether or not the calculation overflows.", ("expression", "An expression returning a set of numeric values."))
+      considerations("Any `null` values are excluded from the calculation.", "`avg(null)` returns `null`.")
       query("MATCH (n:Person) RETURN avg(n.age)", ResultAssertions((r) => {
         r.toList.head("avg(n.age)") should equal(30L)
       })) {
-        p("The average of all the values in the property `age` is returned by the example query.")
+        p("The average of all the values in the property `age` is returned.")
         resultTable()
       }
     }
     section("collect()", "functions-collect") {
       p(
-        """`collect()` collects all the values into a list.
-          |It will ignore `null` values.""".stripMargin)
-      function("collect(expression)", ("expression", "An expression."))
+        """`collect()` returns a list containing the values returned by an expression.
+          |Using this function aggregates data by amalgamating multiple records or values into a single list.""".stripMargin)
+      function("collect(expression)", "A list containing heterogeneous elements; the types of the elements are determined by the values returned by`expression`.", ("expression", "An expression returning a set of values."))
+      considerations("Any `null` values are ignored and will not be added to the list.", "`collect(null)` returns an empty list.")
       query("MATCH (n:Person) RETURN collect(n.age)", ResultAssertions((r) => {
         r.toList.head("collect(n.age)") should equal(Seq(13, 33, 44))
       })) {
-        p("Returns a single row, with all the values collected.")
+        p("All the values are collected and returned in a single list.")
```

Comment:
Removal of the phrase 'single row' moves away from the important point that aggregating functions take many rows and reduce the cardinality of the result set. The new wording could equally well apply to the case where we have a collection in one row, and end with a single list of the collection, still in one row.

Reply:
What I'm trying to say here, and above, is the increased uniformity with which we use the term 'set' or 'list' feels like it is hiding the critical difference between a list existing entirely within one row, and a column, or set of results across many rows. I think this difference needs to be emphasized, not hidden.